### ISSUE TITLE
Cannot add dashlet to dashboard in Suite7/R theme

### ISSUE THREAD

**Issue by simjker** [salesagility/SuiteCRM#2370]
#### Issue
When you use the Suite 7/R theme, if you click the Add Dashlet button on the Home page then the Add pop-up is empty.

#### Expected Behavior
If you click the Add Dashlet button on the Home page then a pop-up should be shown that is populated with the dashlets that can be added to the dashboard.  You should be able to select one of the listed dashlets and it should be added to the currently selected tab on the Home page.

#### Actual Behavior
If you click the Add Dashlet button on the Home page then an empty pop-up is displayed.

#### Possible Fix
The bug was introduced by fix [#2198](https://github.com/salesagility/SuiteCRM/pull/2198)
```
if($('*[id=dashletsList]').length>1) {
 +					dashletsListDiv = $('*[id=dashletsList].modal-body');
 +				}
 +				else {
 +					dashletsListDiv = $('#dashletsList');
 +				}

```
For Suite7 and SuiteP the `else` part of the `if` should be called but it isn't.

#### Steps to Reproduce
[SuiteCRM Demo](http://demo.suiteondemand.com/index.php?action=Login&module=Users&login_module=Home&login_action=index)
1. Login (will,will)
2. Select profile and change theme to Suite7
3. Select Home menu option
4. Click Add Dashlet button

#### Context
This bug prevents the users from customising the dashboard in Suite7 and Suite R.

#### Your Environment
* SuiteCRM Version used: 7.7.5
* Browser name and version:  Chrome Version 53.0.2785.143 m (64-bit)
* Environment name and version : Your demo
* Operating System and version : Your demo

**Comment by simjker** [comment]
Is the test correct?
`$('*[id=dashletsList]').length` = 2 in Suite 7
`$('*[id=dashletsList]').length` = 1 in Suite P

**Comment by staganyi** [comment]
Same issue reported on #2086

**Comment by mayerelyashiv** [comment]
Thank you. It's working on a 7.7.5

**Comment by Doddsy1000** [comment]
I had the same issue.  (Now working well and thanks heaps).
Adding the line to jssource/src_files/include/MySugar/javascript/MySugar.js
as shown at line 502    $('#dashletsList').html(response['html']);    
resolved the issue after restarting the web server.
In my case the repair utilities in SuiteCRM "Quick repair & Rebuild" & "Rebuild JS files" didn't kick this code in.  Restarting Apache also didn't allow the resolution.
Chrome browser also required clearing the history.  Firefox & Edge worked OK.

    CentOS 7 - SuiteCRM 7.7.5
    Apache/2.4.6 (CentOS) PHP/5.6.26
    Database client version: libmysql - 5.6.31
    PHP extension: mysqliDocumentation curlDocumentation mbstringDocumentation
    PHP version: 5.6.26

(P.S. Another test copy of SuiteCRM 7.7.5 remains on the same server. The fix has not been applied to that instance & it still has this dashlet issue until the above code is applied.)

**Comment by bmwtourer** [comment]
I tried the fix, quick rebuild and I still have the problem:
can not add dashlet to home page *unless* you are using SuiteP theme...
SuiteCRM 7.7.5

**Comment by bmwtourer** [comment]
I should add that this problem ALSO occurs with  **a brand new 7.7.5** instance (for testing)

**Comment by bmwtourer** [comment]
I applied the patch and at first it didn't work (as it appeared):
- I performed a quick repair.... no change so I undo my changes...
Then I went to the (new) user who reported the issue and he said that this morning it worked!

Came back to my desk and re-applied the fix and test again:
- no go  same problem
I perform a quick repair
- no go same problem
I log out login, same problem, I close my browser (chrome) and restarted it, same problem...
I login to one terminal server and it was working.
I tried with IE and it was working.
I flushed my chrome cache and it was working 

**So you must clear you browser cache for the modification to take effect.**
